app/src/main/res/layout/rowlayout.xml

```diff
@@ -49,14 +50,15 @@
     <Button
 
         android:id="@+id/btnViewRecipe"
-        android:layout_width="100dp"
+        android:layout_width="52dp"
         android:layout_height="40dp"
-        android:layout_marginLeft="200dp"
+        android:layout_marginLeft="260dp"
         android:layout_marginTop="20dp"
-        android:text="More Info"
         android:background="@android:color/white"
+        android:backgroundTint="#268E2A"
+        android:clickable="true"
         android:textColor="@color/black"
-        android:clickable="true"/>
+        app:icon="@drawable/ic_information" />
```

Comment:
https://cdn.discordapp.com/attachments/512657675016863744/826631301795938365/unknown.png

I notice that the "More Info" text was long in Pixel 2 and 3 makin the recipe detail overlap. Their 2 way I thought of fixing it.
1. Just use a lower case "i" text 
2. Use a information symbol  

Both way are just meant to shorten the length of that button

Reply:
Nice suggestion, I'll implement this change for sure!